tests/backstop_data/engine_scripts/puppet/utility.js

```diff
@@ -103,6 +103,11 @@ module.exports = class CrmPage {
    */
   async waitForAngular () {
     await this.engine.waitForSelector('#bootstrap-theme');
+
+    // remove drupal/civicrm error logging, which creates difference
+    await this.removeElements('#console');
+
+    await this.closeSystemErrorNotification();
```

Comment:
why not remove the notifications instead of closing them and waiting for them to close?